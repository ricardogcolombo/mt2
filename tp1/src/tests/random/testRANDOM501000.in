50 1000
1 17 76 43 45
1 32 37 41 3
1 22 8 16 56
1 26 119 23 27
1 50 72 24 56
1 9 123 38 29
1 20 67 25 86
1 38 64 4 45
1 28 23 13 54
1 48 16 16 61
1 16 78 21 29
1 28 10 50 122
1 25 128 4 103
1 31 108 47 5
1 7 64 39 1
1 8 44 40 67
1 38 111 18 98
1 12 4 47 126
1 41 43 28 101
1 23 12 47 73
1 13 76 17 86
1 34 76 1 4
1 3 80 6 18
1 18 40 15 127
1 15 73 16 49
1 41 50 22 39
1 2 81 29 120
1 48 92 33 36
1 28 39 4 122
1 33 83 3 55
1 39 80 22 68
1 49 128 14 102
1 45 87 46 41
1 16 81 38 2
1 23 51 44 77
1 36 71 25 118
1 3 68 26 12
1 11 44 9 51
1 36 26 23 32
1 42 32 14 77
1 29 84 32 122
1 39 67 6 96
1 32 0 3 100
1 42 126 40 120
1 24 61 7 97
1 37 121 7 47
1 36 11 34 55
1 3 85 42 93
1 17 69 38 118
1 42 16 26 112
1 12 119 21 49
1 44 119 37 83
1 39 33 15 32
1 41 67 23 58
1 46 3 25 102
1 23 85 18 34
1 16 26 24 44
1 14 13 46 20
1 9 99 15 56
1 39 14 47 51
1 7 65 9 46
1 11 100 9 57
1 29 3 23 45
1 27 78 22 51
1 48 108 1 89
1 31 84 11 120
1 19 122 42 82
1 26 92 6 97
1 20 23 1 12
1 12 115 16 123
1 25 119 17 92
1 16 57 33 42
1 23 72 16 7
1 18 90 41 27
1 19 102 1 4
1 37 46 35 4
1 20 57 13 98
1 50 64 28 28
1 7 122 29 113
1 31 126 12 75
1 39 42 20 129
1 3 18 8 66
1 44 69 41 115
1 23 78 22 101
1 16 6 7 109
1 3 0 17 55
1 29 19 23 30
1 2 48 21 57
1 17 59 23 21
1 46 107 36 14
1 2 81 6 32
1 6 66 10 118
1 27 57 7 81
1 27 19 43 13
1 32 45 49 96
1 46 69 1 7
1 12 46 26 70
1 45 40 50 0
1 14 123 35 61
1 29 88 16 41
1 39 61 15 37
1 13 92 48 9
1 32 86 37 26
1 27 112 8 117
1 42 73 16 68
1 41 41 28 43
1 10 36 22 24
1 46 93 38 120
1 39 23 7 81
1 3 31 10 10
1 46 48 10 120
1 30 22 18 51
1 5 52 21 77
1 5 52 9 21
1 39 117 10 76
1 35 108 33 85
1 30 36 42 20
1 48 55 36 12
1 26 41 23 42
1 47 17 25 119
1 1 120 50 127
1 28 11 48 39
1 27 66 13 107
1 15 112 7 47
1 30 1 50 10
1 19 116 33 54
1 27 105 29 78
1 32 52 35 27
1 45 97 29 108
1 25 54 28 16
1 30 73 16 21
1 47 120 37 127
1 41 75 9 84
1 50 43 30 63
1 43 98 36 13
1 46 87 47 47
1 34 30 29 92
1 43 105 5 86
1 42 86 38 64
1 4 7 24 91
1 4 113 10 48
1 23 4 29 95
1 43 23 34 18
1 1 71 24 111
1 27 87 6 23
1 1 29 5 95
1 23 130 24 75
1 16 101 50 47
1 50 63 6 83
1 4 78 40 23
1 47 45 25 61
1 10 12 19 27
1 1 111 37 120
1 3 115 27 70
1 10 62 32 18
1 12 90 40 8
1 24 26 39 41
1 41 4 50 37
1 44 19 20 124
1 33 7 23 114
1 20 76 5 84
1 5 126 47 122
1 46 93 7 65
1 37 7 12 52
1 37 108 21 48
1 47 0 11 75
1 38 17 11 21
1 2 41 24 75
1 13 32 14 101
1 37 109 42 49
1 40 89 12 97
1 23 27 8 12
1 6 55 35 82
1 48 28 34 57
1 37 66 4 88
1 2 17 35 100
1 25 129 37 90
1 33 34 26 43
1 17 22 21 123
1 44 69 10 67
1 16 10 19 25
1 13 97 27 19
1 4 25 15 122
1 48 43 10 60
1 27 73 17 124
1 2 25 9 108
1 50 16 19 62
1 17 93 4 71
1 11 118 38 91
1 48 106 16 100
1 50 104 21 106
1 16 9 47 99
1 25 93 10 117
1 12 55 7 13
1 38 121 18 47
1 45 28 17 129
1 50 56 43 126
1 13 12 41 42
1 46 77 40 52
1 28 118 49 84
1 45 112 19 48
1 7 99 18 89
1 12 1 9 0
1 39 128 14 108
1 31 39 24 128
1 7 32 29 114
1 25 126 12 80
1 39 59 43 6
1 12 96 25 45
1 49 27 19 55
1 36 84 6 78
1 22 13 44 5
1 5 6 1 123
1 14 41 35 32
1 41 53 48 10
1 37 96 39 118
1 44 51 7 58
1 11 12 42 41
1 37 110 50 26
1 40 38 28 65
1 26 0 46 101
1 17 46 30 69
1 2 58 21 51
1 24 72 8 94
1 46 50 5 5
1 13 86 1 18
1 27 125 4 88
1 5 69 27 115
1 46 38 10 34
1 29 74 44 23
1 41 25 1 106
1 17 24 16 87
1 37 110 6 99
1 38 11 15 100
1 44 13 28 18
1 37 17 10 85
1 29 65 38 52
1 18 91 42 25
1 12 51 16 33
1 7 19 6 90
1 13 60 27 27
1 38 130 18 26
1 26 81 23 74
1 48 73 9 43
1 29 122 39 87
1 47 25 48 113
1 17 52 46 24
1 38 44 6 76
1 8 51 34 88
1 27 114 32 39
1 36 42 42 43
1 23 108 46 23
1 17 76 45 6
1 50 34 47 62
1 36 30 38 8
1 3 78 20 101
1 6 28 47 73
1 18 49 5 47
1 44 12 1 46
1 42 130 35 103
1 39 105 21 115
1 49 37 41 72
1 6 77 8 78
1 44 7 26 100
1 13 50 39 92
1 25 88 27 90
1 14 101 44 105
1 19 40 42 112
1 11 88 35 69
1 17 36 40 81
1 46 100 45 82
1 48 22 42 5
1 33 104 16 77
1 20 7 16 51
1 43 53 48 70
1 12 79 44 6
1 22 118 49 23
1 8 112 1 29
1 34 68 36 52
1 2 0 50 114
1 11 41 2 9
1 12 49 6 13
1 28 129 29 119
1 48 107 39 18
1 7 57 43 69
1 40 114 22 30
1 14 23 41 42
1 6 78 5 52
1 6 72 3 127
1 1 87 8 22
1 46 52 23 89
1 1 18 43 127
1 32 56 3 94
1 48 1 4 88
1 14 99 48 43
1 49 39 7 119
1 23 89 33 109
1 35 37 31 18
1 10 16 24 23
1 46 82 27 109
1 29 64 27 46
1 31 109 11 80
1 38 66 2 115
1 10 50 18 67
1 3 80 44 6
1 7 129 20 33
1 32 98 36 84
1 2 13 42 112
1 13 129 32 55
1 25 99 21 58
1 43 39 17 74
1 9 129 4 86
1 40 71 5 81
1 44 67 19 85
1 32 94 16 119
1 10 98 12 90
1 19 58 49 114
1 18 90 46 70
1 32 34 26 59
1 27 68 14 55
1 20 102 43 83
1 1 44 48 33
1 14 31 45 80
1 8 33 42 78
1 33 39 26 13
1 20 44 38 130
1 32 75 45 112
1 39 60 25 37
1 31 96 22 16
1 34 87 28 26
1 16 18 12 36
1 39 53 24 33
1 9 62 42 86
1 42 12 24 110
1 35 71 31 44
1 50 4 15 99
1 48 10 5 30
1 43 92 24 33
1 5 119 45 76
1 11 113 38 14
1 48 126 19 92
1 14 29 13 16
1 41 17 1 114
1 37 116 38 4
1 15 126 20 1
1 49 39 32 127
1 10 61 4 77
1 43 30 13 4
1 47 19 48 0
1 26 68 1 79
1 23 34 44 71
1 43 105 32 16
1 43 103 24 43
1 13 116 35 45
1 16 127 15 96
1 34 75 13 63
1 32 85 35 81
1 45 83 6 10
1 41 31 42 102
1 22 56 1 84
1 20 122 11 77
1 17 46 14 32
1 5 17 44 85
1 36 4 21 23
1 49 12 15 104
1 28 23 11 103
1 48 114 32 62
1 42 18 48 79
1 11 126 38 77
1 11 24 35 18
1 47 54 41 129
1 16 85 35 115
1 7 75 8 105
1 48 96 20 21
1 14 70 45 62
1 22 55 37 97
1 33 48 39 55
1 3 125 1 71
1 10 83 30 41
1 11 18 17 63
1 18 55 12 79
1 16 81 38 120
1 12 75 32 82
1 26 34 10 116
1 25 29 40 127
1 46 18 29 110
1 48 130 37 117
1 27 43 4 52
1 38 72 44 28
1 8 76 31 120
1 25 45 48 128
1 30 91 12 75
1 8 46 22 110
1 5 8 25 111
1 50 56 16 72
1 23 46 4 12
1 44 130 15 16
1 47 40 25 93
1 46 58 11 53
1 38 105 34 9
1 44 67 47 77
1 46 6 32 87
1 17 4 16 11
1 28 64 5 24
1 38 113 19 53
1 9 48 43 70
1 32 40 38 36
1 21 99 22 128
1 11 41 43 103
1 45 44 2 85
1 28 50 11 129
1 15 91 31 13
1 17 111 20 109
1 32 56 19 24
1 25 41 3 123
1 12 87 37 18
1 27 112 23 43
1 39 66 41 0
1 25 55 38 9
1 12 37 40 43
1 23 126 22 81
1 35 122 18 98
1 36 111 19 81
1 16 48 31 0
1 7 47 28 114
1 49 7 11 16
1 41 49 28 74
1 38 34 10 60
1 18 58 39 66
1 14 63 34 50
1 10 74 28 1
1 47 63 21 98
1 28 121 21 109
1 45 103 6 37
1 25 113 4 11
1 45 108 13 9
1 31 0 17 1
1 15 121 43 52
1 16 87 41 75
1 15 42 3 80
1 22 31 13 46
1 47 65 30 8
1 20 126 29 6
1 49 41 12 34
1 49 51 10 42
1 33 43 35 119
1 31 47 46 97
1 17 41 40 11
1 16 94 14 119
1 11 8 26 48
1 18 92 31 12
1 11 82 34 120
1 16 14 9 67
1 23 50 30 87
1 46 27 8 40
1 25 15 41 58
1 34 22 39 3
1 38 7 3 22
1 44 68 17 107
1 26 100 42 112
1 39 89 24 71
1 10 15 22 46
1 38 61 41 93
1 19 13 40 126
1 12 21 25 118
1 28 128 50 129
1 39 99 11 39
1 25 109 15 62
1 31 93 3 89
1 9 12 50 85
1 9 72 39 92
1 46 124 28 116
1 9 89 3 45
1 6 13 27 90
1 23 127 18 66
1 5 90 30 78
1 33 64 40 124
1 20 18 23 7
1 30 104 20 1
1 26 23 43 118
1 34 29 21 52
1 45 100 25 19
1 21 49 48 77
1 4 129 25 128
1 16 63 9 39
1 20 110 33 126
1 22 34 23 93
1 31 121 32 75
1 14 96 5 126
1 49 124 5 38
1 29 48 38 115
1 45 15 47 19
1 45 66 42 59
1 32 14 45 25
1 44 60 11 42
1 35 91 36 71
1 37 40 8 68
1 41 14 15 104
1 32 116 46 34
1 10 71 28 22
1 5 27 31 130
1 1 54 4 40
1 7 122 2 86
1 3 115 40 34
1 22 36 8 101
1 32 8 23 114
1 5 80 4 85
1 19 55 39 99
1 19 119 37 92
1 32 101 19 111
1 22 68 12 59
1 28 94 22 95
1 33 50 41 94
1 27 67 22 0
1 7 67 34 120
1 24 111 1 1
1 38 47 30 10
1 17 25 48 4
1 28 115 4 2
1 7 22 34 52
1 3 77 2 19
1 13 46 15 42
1 13 114 46 88
1 17 126 39 2
1 44 101 36 40
1 3 85 42 36
1 32 45 16 109
1 42 12 8 25
1 35 87 31 84
1 31 41 45 86
1 48 102 17 76
1 41 88 4 102
1 40 101 12 121
1 4 121 24 76
1 45 48 42 77
1 11 20 36 86
1 32 25 47 116
1 10 93 35 78
1 7 99 17 72
1 31 129 33 56
1 20 14 34 57
1 9 20 1 67
1 13 24 24 113
1 25 122 35 80
1 19 84 46 7
1 37 51 15 67
1 29 34 39 59
1 1 38 8 74
1 14 116 6 84
1 43 31 50 24
1 19 80 12 77
1 23 103 25 111
1 34 9 17 108
1 36 108 30 17
1 37 83 35 24
1 48 64 19 7
1 7 121 27 67
1 29 18 14 51
1 18 43 31 115
1 26 36 21 57
1 47 42 27 50
1 13 65 47 19
1 38 82 39 87
1 29 121 35 48
1 48 46 15 129
1 40 105 5 20
1 11 58 4 31
1 35 5 16 93
1 8 2 27 20
1 9 23 23 53
1 29 20 50 86
1 7 7 11 89
1 37 83 38 28
1 36 83 28 28
1 47 95 35 42
1 12 59 47 18
1 13 125 22 5
1 38 90 49 126
1 33 30 46 51
1 3 39 12 117
1 6 24 13 3
1 12 19 3 49
1 41 47 15 81
1 46 127 2 128
1 46 112 49 105
1 14 114 10 73
1 10 111 13 27
1 2 58 48 91
1 29 130 25 38
1 27 81 40 8
1 9 120 38 71
1 22 129 32 53
1 29 10 3 91
1 39 100 11 28
1 5 48 41 89
1 9 116 31 106
1 28 36 22 74
1 27 111 29 58
1 10 74 14 122
1 30 37 19 31
1 42 106 25 14
1 45 45 5 57
1 49 129 34 67
1 39 45 4 43
1 32 38 19 128
1 1 98 23 51
1 36 83 2 100
1 37 91 21 64
1 34 110 2 8
1 17 28 48 53
1 47 71 28 48
1 45 67 32 43
1 28 58 35 52
1 50 38 6 33
1 21 127 14 112
1 31 116 30 108
1 17 70 3 86
1 29 66 8 40
1 28 103 13 115
1 15 29 29 74
1 44 1 18 73
1 36 89 3 25
1 18 38 14 129
1 10 107 13 112
1 34 27 26 45
1 26 80 11 63
1 1 18 12 130
1 33 34 28 21
1 27 34 35 32
1 43 25 7 40
1 11 46 42 129
1 19 82 43 91
1 11 26 17 68
1 28 33 8 14
1 50 60 4 40
1 38 97 14 68
1 16 129 15 83
1 11 73 9 58
1 28 29 25 11
1 11 41 2 55
1 5 44 25 34
1 20 23 30 98
1 32 63 15 49
1 8 83 31 59
1 49 20 4 12
1 31 24 32 8
1 43 22 11 29
1 48 92 17 24
1 48 69 46 50
1 6 104 16 73
1 10 115 4 102
1 45 52 21 77
1 32 87 12 71
1 49 107 29 122
1 34 89 49 71
1 28 93 41 128
1 1 18 28 120
1 36 124 48 61
1 16 67 32 61
1 6 75 20 25
1 38 67 13 88
1 9 54 23 13
1 25 31 49 98
1 43 83 49 75
1 48 41 8 58
1 27 60 2 110
1 2 38 28 112
1 7 129 47 52
1 8 87 14 73
1 16 37 5 83
1 10 67 8 54
1 29 41 31 119
1 31 35 29 2
1 29 55 47 70
1 18 74 24 13
1 40 111 4 76
1 39 126 10 127
1 38 41 43 36
1 19 81 21 117
1 36 71 48 118
1 13 117 26 82
1 25 106 32 126
1 1 87 24 20
1 35 48 25 81
1 22 46 1 53
1 5 2 19 62
1 19 0 40 59
1 3 81 33 32
1 1 85 44 79
1 29 49 17 35
1 35 101 11 22
1 17 37 13 35
1 14 10 34 96
1 47 53 25 19
1 1 31 15 50
1 1 8 32 90
1 20 84 49 21
1 42 122 36 32
1 46 77 9 83
1 48 113 34 38
1 34 68 42 56
1 33 58 15 84
1 30 50 39 34
1 30 111 20 108
1 2 97 33 103
1 27 95 38 50
1 1 10 30 104
1 33 34 5 18
1 2 127 7 63
1 23 62 46 60
1 33 74 40 91
1 28 6 37 30
1 24 114 8 107
1 27 82 34 58
1 7 80 8 42
1 27 69 42 129
1 19 91 35 66
1 42 54 29 61
1 35 44 31 62
1 27 49 9 72
1 16 26 18 69
1 12 77 1 99
1 21 31 27 54
1 36 119 2 17
1 13 1 11 73
1 43 69 45 10
1 10 1 27 22
1 29 68 6 74
1 40 14 37 16
1 19 97 10 1
1 37 58 34 84
1 29 56 5 118
1 20 127 47 32
1 44 44 6 82
1 5 100 38 44
1 16 51 23 46
1 40 60 31 76
1 3 101 33 41
1 23 48 29 109
1 15 59 9 121
1 2 123 33 19
1 8 82 14 111
1 16 107 25 102
1 49 0 40 38
1 4 39 16 64
1 39 128 15 12
1 30 10 24 18
1 17 81 45 80
1 23 80 47 65
1 40 38 24 122
1 44 15 40 3
1 36 112 39 13
1 41 5 45 100
1 16 4 14 45
1 8 3 17 84
1 12 106 38 56
1 49 112 42 99
1 19 25 31 45
1 49 40 7 118
1 7 26 41 90
1 49 108 8 66
1 7 120 28 62
1 19 40 35 3
1 13 101 28 118
1 42 16 10 72
1 16 6 8 71
1 1 72 47 22
1 14 19 15 49
1 26 69 16 37
1 16 75 3 10
1 27 111 15 34
1 32 122 28 75
1 19 61 1 66
1 17 122 2 3
1 7 29 50 87
1 4 56 10 95
1 7 84 18 120
1 19 117 35 43
1 36 119 8 126
1 22 37 27 2
1 45 40 18 57
1 22 107 6 125
1 10 44 8 1
1 32 113 13 38
1 39 90 23 19
1 18 66 36 53
1 34 89 23 64
1 14 0 41 3
1 34 5 10 17
1 46 110 6 130
1 43 75 4 0
1 43 87 8 53
1 12 78 10 102
1 31 102 16 107
1 16 16 49 55
1 30 33 10 40
1 15 122 1 86
1 36 61 41 127
1 39 66 4 47
1 37 103 29 66
1 12 49 43 98
1 20 125 43 120
1 44 39 16 61
1 32 72 24 109
1 21 61 27 129
1 41 21 43 43
1 34 119 26 60
1 31 64 29 80
1 46 20 5 25
1 5 60 38 28
1 48 50 8 32
1 24 39 50 114
1 24 100 41 53
1 50 73 21 5
1 28 85 21 25
1 34 117 48 93
1 19 91 17 10
1 12 97 23 106
1 14 75 11 111
1 21 129 11 26
1 32 88 34 13
1 26 89 21 65
1 47 55 34 123
1 29 58 46 115
1 45 29 32 109
1 44 55 15 67
1 1 0 7 101
1 17 97 38 53
1 50 44 33 52
1 35 32 10 26
1 44 130 38 78
1 3 8 20 68
1 9 116 50 28
1 48 31 37 130
1 13 48 1 106
1 26 114 49 82
1 20 43 40 45
1 1 58 18 59
1 18 98 17 54
1 23 39 18 11
1 42 26 18 124
1 28 109 13 1
1 8 106 38 28
1 27 113 49 44
1 46 114 15 68
1 48 10 2 130
1 36 61 8 20
1 20 44 29 67
1 8 28 29 50
1 13 86 19 102
1 47 36 38 26
1 50 60 19 72
1 27 35 6 22
1 11 39 19 91
1 40 113 7 81
1 28 114 30 88
1 4 82 12 23
1 44 126 9 90
1 50 95 38 78
1 12 107 15 47
1 29 45 44 117
1 41 107 27 129
1 30 55 4 121
1 30 96 44 35
1 39 74 20 18
1 18 121 44 126
1 31 84 50 106
1 11 25 32 9
1 13 84 36 121
1 13 81 6 122
1 30 119 43 124
1 22 115 18 75
1 31 64 6 104
1 5 1 49 114
1 26 32 7 80
1 17 126 21 33
1 43 105 22 121
1 23 116 14 98
1 40 74 32 4
1 26 78 30 128
1 1 71 4 12
1 7 10 47 117
1 10 89 18 112
1 7 71 23 8
1 23 38 22 129
1 15 0 1 108
1 15 53 6 86
1 36 77 11 4
1 22 59 47 10
1 19 37 9 53
1 26 68 8 8
1 24 117 30 28
1 48 109 13 80
1 22 58 26 14
1 39 109 23 111
1 8 14 5 120
1 25 58 49 77
1 43 92 24 110
1 6 129 42 33
1 41 85 19 118
1 38 84 13 104
1 42 111 32 44
1 20 80 17 77
1 47 101 37 30
1 14 111 16 36
1 30 14 14 76
1 31 127 1 120
1 33 94 19 128
1 12 100 29 95
1 31 128 18 0
1 1 32 39 73
1 7 61 25 15
1 23 22 5 58
1 14 120 6 113
1 48 75 15 25
1 9 21 10 9
1 50 107 7 38
1 30 21 22 23
1 39 63 34 102
1 40 104 20 23
1 46 97 1 62
1 38 51 2 62
1 4 34 32 94
1 7 117 35 38
1 30 53 45 123
1 15 100 37 102
1 26 91 9 30
1 13 76 30 5
1 30 19 21 27
1 29 118 20 36
1 44 114 18 115
1 28 60 9 35
1 38 28 18 70
1 25 77 36 37
1 50 38 33 125
1 2 116 35 107
1 48 77 1 124
1 37 77 31 10
1 31 9 32 77
1 50 62 9 28
1 41 129 32 55
1 36 27 1 126
1 41 88 12 22
1 44 93 38 4
1 29 76 27 20
1 33 111 18 119
1 5 41 4 123
1 35 122 13 43
1 31 47 20 92
1 36 67 44 55
1 33 122 7 5
1 50 61 7 94
1 17 69 24 34
1 25 52 10 35
1 20 51 11 72
1 35 79 6 106
1 28 116 44 29
1 45 107 28 125
1 9 69 30 49
1 41 6 14 62
1 24 114 6 46
1 28 117 15 4
1 16 50 34 75
1 49 8 45 129
1 22 119 5 17
1 34 71 13 51
1 32 11 34 1
1 36 85 21 71
1 17 61 40 33
1 23 36 24 15
1 4 69 44 91
1 3 24 36 32
1 34 71 50 97
1 37 72 33 94
1 36 89 7 82
1 25 110 1 82
1 7 75 29 111
1 6 5 9 95
1 29 13 3 101
1 24 113 22 39
1 44 76 19 120
1 20 56 14 76
1 2 109 35 123
1 20 122 10 17
1 19 86 21 84
1 14 73 41 60
1 33 40 13 112
1 27 108 21 95
1 15 113 26 90
1 41 14 46 18
1 17 66 44 68
1 32 64 50 35
1 41 88 48 41
1 27 93 8 80
1 40 122 7 51
1 2 21 47 110
1 22 38 13 123
1 31 71 4 37
1 5 67 12 103
1 31 32 7 80
