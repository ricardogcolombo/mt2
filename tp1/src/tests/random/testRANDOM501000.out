0.454538
0.415622
0.611815
0.380130
0.445648
0.480702
0.540304
0.442271
0.521110
0.525844
0.482328
0.497965
0.498952
0.554256
0.494285
0.492311
0.626884
0.523973
0.583545
0.426275
0.469979
0.493995
0.510160
0.550250
0.521152
0.483615
0.644945
0.493765
0.525188
0.415177
0.472930
0.478283
0.388968
0.639483
0.365915
0.466458
0.541094
0.306642
0.641359
0.514929
0.460743
0.521137
0.628854
0.410632
0.506676
0.473231
0.487433
0.602298
0.464749
0.521204
